src/schemas/movies.py

```
@@ -0,0 +1,154 @@
+from datetime import date, datetime
+from typing import Optional, List
+
+from pydantic import BaseModel, Field, field_validator
+
+from database.models.movies import MovieStatusEnum
+from schemas.examples.movies import (
+    country_schema_example,
```

Ruff (0.11.9)

`schemas.examples.movies.country_schema_example` imported but unused

Remove unused import

(F401)